### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by Baars-IT**
Unit mapping file:
- I added validation before it is 'imported'. It contained some validation errors.
- Easiest to have entries in (kind of) alphabetical order

### CODE REVIEW

File: DataFiles/unit-mapping.csv [hunk 1]
```diff
@@ -59,8 +62,10 @@
 "mm/(m⋅K)", "mm/(m·K)"
 "mNHN", "?"
 "MN/m2", "MN/m²"
+"m/Nm²", "m/(N·m²)"
 "MN/m3", "MN/m³"
-"(m².K)/W", "(m²·K)/W"
+"(m².K)/W", "m²·K/W"
+"(m²·K)/W", "m²·K/W"
```
Comment (atomczak) [inline DataFiles/unit-mapping.csv:68]: duplicate

Comment (atomczak) [reply DataFiles/unit-mapping.csv:68]: I now see they are not :)

**Review by atomczak — APPROVED** [review]
good. I added some changes to GraphQL description

File: DataFiles/unit-mapping.csv (hunk 1 above)
Comment (Erik-RN) [reply DataFiles/unit-mapping.csv:68]: Yep :-). That's why I added validation of this file before uploading it to bSDD, to prevent duplicates

**Pull request merged**